docs/src/main/sphinx/connector/prometheus.md

```diff
@@ -91,6 +91,14 @@ The following configuration properties are available:
      with the values as `value1` and `value2`. Escape comma (`,`) or colon(`:`)
      characters in a header name or value with a backslash (`\`).
   -
+* - `prometheus.query.match.string`
+  -  Match string to send as part of query to Prometheus to filter the data on Prometheus server.
+     The equivalent catalog session property is `query_match_string`.
+  -
+* - `prometheus.query.functions`
+  -  Comma separated list of functions to be sent to Prometheus HTTP API as part of query.
+     The equivalent catalog session property is `query_functions`.
+  -
```

Comment:
I don't think using session properties which affect query results is a good idea. We usually don't do that. 
Please consider creating a table function instead: https://trino.io/docs/current/develop/table-functions.html

Reply:
Thank you @ebyhr , closing this PR, will create a new one with Table Function